global_planner/cfg/GlobalPlanner.cfg

```diff
@@ -10,4 +10,14 @@ gen.add("neutral_cost", int_t,   0, "Neutral Cost",  50, 1, 255)
 gen.add("cost_factor", double_t, 0, "Factor to multiply each cost from costmap by", 3.0, 0.01, 5.0)
 gen.add("publish_potential", bool_t, 0, "Publish Potential Costmap", True)
 
+orientation_enum = gen.enum([ 
+    gen.const("None",        int_t, 0, "No orientations added except goal orientation"),
+    gen.const("Forward",     int_t, 1, "Orientations point to the next point on the path"),
+    gen.const("Interpolate", int_t, 2, "Orientations are a linear blend of start and goal pose"),
+    gen.const("Mixture",     int_t, 3, "Forward orientation until last straightaway"),
```

Comment:
Is there a better name than "Mixture"? (or is this based off some paper where that name was used -- if so, we should include the link to said paper)